pages/01_embedding_and_jsapi.md

```diff
@@ -43,29 +70,30 @@ function initializeViz() {
 
 **Custom interfaces for interacting with the view** -- Dashboards often have elements for filtering, changing parameters, and switching tabs, amongst other things. When integrating with another application, you may want to match the look and feel of dashboard interfaces with the look and feel of the embedding application. For example, you can create a drop-down with HTML/CSS/JS that matches the style of your application and have that drop-down make JavaScript API calls to filter the viz or change a parameter.
 
 **Custom interactions** -- Because the JavaScript API gives you programmatic control over interaction, you can combine calls to create interactions that would not be possible otherwise. For example, you could create a button that switches to a specific visualization, changes a parameter, and then selects a set of marks all with a single click by the user. Or, when a user clicks a mark on a viz, you could have that drive a parameter change on another sheet. The combination of API calls is limited only by your imagination.
 
-**Integration with other systems** -- If a visualization is embedded inside another system, and a user finds an insight from a visualization, it is likely the user will want to take action on that insight elsewhere in your application. With the JavaScript API, you can query data on the visualization, or listen to user events, and execute code that takes the appropriate action.
+**Integration with other systems** -- If a visualization is embedded inside another system, and a user finds an insight from a visualization, it is likely the user will want to take action on that insight elsewhere in your application. With the JavaScript API and Embedding API, you can query data on the visualization, or listen to user events, and execute code that takes the appropriate action.
 
 In the image below, an HR-focused dashboard allows the user to select employees for an audit. When the user is ready, he or she can click "Run Audit" to generate a non-Tableau audit based on the selections made inside the Tableau viz.
 
 ![Embed Code]({{ site.baseurl }}/img/run_audit.png)
 
-## JavaScript API Resources
+## Embedding API and JavaScript API Resources
 
 Link | Description
 ---- | -----------
-[JavaScript API Documentation](https://onlinehelp.tableau.com/current/api/js_api/en-us/JavaScriptAPI/js_api.htm#) | Official knowledge-base for the JavaScript API including an overview, tutorial, samples, explanation of key concepts, and API reference
-[JavaScript API Tutorial](https://onlinehelp.tableau.com/samples/en-us/js_api/tutorial.htm) | An interactive tutorial that will walk you through the key concepts of the JavaScript API
+[Embedding API v3 Documentaion](https://help.tableau.com/current/api/embedding_api/en-us/index.html) | Official documentation for the Embedding API v3, including overview, key concepts, and API reference
```

Comment:
Do we want to include the Developer Preview here too or as a separate section?

Reply:
@illonage  We could add something here and mention the Embedding API v3 preview specifically. Although, we'd have to remember to update it when the preview is over.  Or we could just provide a link the program and tell something general about previews in general.

Reply:
I added the Developer Program as a subsection to the EAPIv3 section, and then added a link to the table at the bottom of the page.